app/src/main/java/uk/gov/onelogin/appinfo/service/data/AppInfoServiceImpl.kt

```diff
@@ -34,7 +43,9 @@ class AppInfoServiceImpl @Inject constructor(
         fallback: AppInfoServiceState
     ): AppInfoServiceState {
         return if (localSourceState is AppInfoLocalState.Success) {
-            AppInfoServiceState.LocalSuccess(localSourceState.value)
+            // Check local min version compatible
+            appVersionCheck.compareVersions(localSourceState.value)
```

Comment:
Can we just return this line? Given its under local success already it _should_ give a success back anyway